src/main/res/values/strings.xml

```diff
@@ -161,12 +161,12 @@
     <string name="unverified">Unverified</string>
     <string name="update">Update</string>
     <string name="updates">Updates</string>
-    <string name="upstream_source_code_is_not_free">Upstream source code is not free</string>
+    <string name="upstream_source_code_is_not_free">The upstream source code is not free</string>
     <string name="username">Username</string>
     <string name="username_missing">Username missing</string>
     <string name="validation_index_error_DESC">Index could not be validated.</string>
     <string name="version">Version</string>
-    <string name="version_FORMAT">Version %s</string>
+    <string name="version_FORMAT">%s version</string>
```

Comment:
Are you proposing changing "version 1.2" to "1.2 version"? If so, that is incorrect.

Reply:
I don't know what happened there. Good catch :)